packages/eslint-plugin-warp-drive/src/rules/no-create-record-rerender.js

```diff
@@ -0,0 +1,164 @@
+// @ts-check
+
+const messageId = 'noCreateRecordRerender';
+const createRecordExpression = 'CallExpression[callee.property.name="createRecord"]';
+const emberObjectExtendExpression = 'CallExpression[callee.property.name="extend"]';
+const forbiddenParentMethodKeyNames = [
+  'init',
+  'initRecord', // legacy modals
```

Comment:
IIRC this is AB-specific.

Reply:
yeah, may need to add config to allow additional methods to be disallowed